lib/releasecop/manifest_item.rb

```diff
@@ -0,0 +1,28 @@
+class ManifestItem
+  attr_reader :git, :branch, :name, :hokusai_tag
+
+  def initialize(repo_name, item)
+    @repo_name = repo_name
+    @git = item['git']
+    @name = item['name']
+    @branch = item['branch'] || 'master'
+    @hokusai_tag = item['hokusai']
+  end
+
+  def for_rev_range
+    sha = find_hokusai_sha if @hokusai_tag
```

Comment:
This is an expensive/slow operation and shouldn't be run each time `for_rev_range` is called. Instead it can be memoized or just set in the constructor.

Reply:
i actually had it in constructor originally but the issue is `git clone` of the repo happens few lines after creating `ManifestItem`s.  Looking at it now again maybe i can improve that part otherwise i'll memoize.